lib/features/modeling/cmd/ReconnectConnectionHandler.js

```diff
@@ -11,18 +11,13 @@ export default function ReconnectConnectionHandler(modeling) {
 ReconnectConnectionHandler.$inject = [ 'modeling' ];
 
 ReconnectConnectionHandler.prototype.execute = function(context) {
-
   var newSource = context.newSource,
       newTarget = context.newTarget,
       connection = context.connection,
       dockingOrPoints = context.dockingOrPoints;
 
   if (!newSource && !newTarget) {
-    throw new Error('newSource or newTarget are required');
-  }
-
-  if (newSource && newTarget) {
-    throw new Error('must specify either newSource or newTarget');
+    throw new Error('new source or target is required');
```

Comment:
`new source` -> `newSource`, `target` => `newTarget` would be better names in my point of view. Helps you with a proper type hint.